src/components/partner/partner.service.ts

```diff
@@ -141,6 +142,15 @@ export class PartnerService {
       );
     }
 
+    if (languagesOfConsulting) {
+      await this.repo.updateRelationList({
+        id: object.id,
+        relation: 'languagesOfConsulting',
+        label: 'Language',
+        newList: languagesOfConsulting,
+      });
+    }
```

Comment:
Now that Carson is handling the validation within `updateRelationList`, you can handle the exceptions as shown [in this example](https://github.com/SeedCompany/cord-api-v3/blob/develop/src/components/partner/partner.service.ts#L153C2-L158C2).